src/Emoji.php

```diff
@@ -5778,6 +5778,18 @@ public static function getCharacter(string $characterName) : string
         return constant('static::'.$constantName);
     }
 
+    public static function getCountryFlag(string $countryCode) : string
```

Comment:
Let's make this a bit shorter. What do you think about renaming this to `countryFlag`?